tests/std/tests/P1614R2_spaceship/test.cpp

```diff
@@ -28,21 +29,23 @@
 #include <vector>
 
 template <class T>
-using SpaceshipType = decltype(std::declval<T>() <=> std::declval<T>());
+using SpaceshipType = std::compare_three_way_result_t<T, T>;

```

Comment:
The second parameter of `compare_three_way_result_t` defaults to the first, so you could actually replace uses of `SpaceshipType` herein with `std::compare_three_way_result_t`. (Or just drop the `, T` if there's a reason to keep `SpaceshipType` around.)